providerserver/providerserver.go

```diff
@@ -0,0 +1,59 @@
+package providerserver
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-framework/tfsdk"
+	"github.com/hashicorp/terraform-plugin-go/tfprotov6"
+	"github.com/hashicorp/terraform-plugin-go/tfprotov6/tf6server"
+)
+
+// Returns a protocol version 6 ProviderServer implementation based on the
```

Comment:
Perhaps prefix with `NewProtocol6 `?

Reply:
Updated in [81cba2c](https://github.com/hashicorp/terraform-plugin-framework/pull/308/commits/81cba2c2c215b865eeafd7f013ebed402c5b05a5) -- we should figure out what linter is triggering the warnings so we can either explicitly enable or disable it per team preference.